### PR TITLE
[refactor] kubectl changes to go with those submitted in kraken-lib

### PR DESCRIPTION
**Pull request by alejandroEsc**
**What this PR does / why we need it**: Refactors code to the changes in line with changes in https://github.com/samsung-cnct/kraken-lib/pull/854. Allows for deterministic patters of input and correct handling of arguments passed. 

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```NONE
```

### CODE REVIEW

**Comment by coffeepac** [comment]
going to dismiss @joejulian's review as he is blocking on my review.

File: cmd/cluster_down.go [hunk 1]
```diff
@@ -26,9 +26,9 @@ var downtagsList string
 
 // downCmd represents the down command
 var downCmd = &cobra.Command{
-	Use:           "down [path to Kraken config file]",
+	Use:           "down ",
```
Comment (joejulian) [inline cmd/cluster_down.go]: extra space in the string

File: cmd/cluster_info.go [hunk 2]
```diff
@@ -22,13 +22,12 @@ import (
 
 // infoCmd represents the info command
 var infoCmd = &cobra.Command{
-	Use:           "info",
-	Short:         "Print out cluster state information",
+	Use:   "info",
```
Comment (joejulian) [inline cmd/cluster_info.go]: Do we want to keep with the equally indented strings just for consistency?

File: cmd/cluster_info.go [hunk 3]
```diff
@@ -22,13 +22,12 @@ import (
...
+	Short: "Print out cluster state information",
+	Long: "Output some basic information on the current	cluster state configured by the specified Krakenlib yaml",
```
Comment (joejulian) [inline cmd/cluster_info.go]: extra whitespace in the string

**Pull request merged**